devtools-ci-index.yaml

```diff
@@ -2918,7 +2920,7 @@
             saas_git: saas-errortracking
             prj_name: sentry-preview
             timeout: '20m'
-        - '{ci_project}-{git_repo}-f8planner-snapshot':
```

Comment:
this will need to then be manually removed from the cico side of things, lets make sure we co-ordinate the merge

Reply:
this is not a must have but I wanted to avoid `snapshot` word from the name of the job, It can survive with it as well.
But why does it need cico side changes to rename a job?

Reply:
you cant rename jobs, the jenkins side will see the rendered output  - and only the rendered output, not a delta. So it just see's a new job, and creates it. It will not touch jobs it no longer knows anything about and you end up with the orphaned job without the -snapshot name, but it still has all the access rights, all the deployment bits etc and they will continue to run.

in effect, there is no rename jobs or a delete jobs option. Anything that needs those actions needs to be manually co-ordinated on the jenkins side ( which isnt a problem, just needs to be done that way ).